README.md

```diff
@@ -36,6 +37,13 @@ Here you will find all resources which will be given out throughout your time he
 * Golden Gate Instructors 
     * golden-gate-instructors@appacademy.io
 
+## Cohort Calendar
+* Your Cohort's Calendar is found on the README of your Cohort's Branch
+* To get there, go to the top of the page and click the branch dropdown 
```

Comment:
"to get to your branch, go to the top of the page, and click the branch dropdown. This process is demonstrated in the image below". Maybe we can reword it to be this so that we explicitly say what the image is demonstrating.